.github/workflows/continuous-integration.yml

```diff
@@ -1232,3 +1232,47 @@ jobs:
           channel_id: ${{ env.VETS_WEBSITE_CHANNEL_ID }}
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+  # Please keep steps-evaluation at the end of the workflow as it's meant to evaluate the entirety of the workflow steps that have run.
+  steps_evaluation:
+    runs-on: ubuntu-latest
+    name: Steps Evaluation
+    needs:
+      [
+        'build',
+        'security-audit',
+        'linting',
+        'cypress-tests',
+        'testing-reports-cypress',
+        'archive',
+      ]
+    if: ${{ always() }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-gov-west-1
+      - name: Get va-vsp-bot token
+        uses: marvinpinto/action-inject-ssm-secrets@v1.2.1
+        with:
+          ssm_parameter: /devops/VA_VSP_BOT_GITHUB_TOKEN
+          env_variable_name: VA_VSP_BOT_GITHUB_TOKEN
+
+      - name: Init Dashboard Data Repo
+        uses: department-of-veterans-affairs/platform-release-tools-actions/init-data-repo@main
+
+      - name: Set Up BigQuery Creds
+        uses: ./.github/workflows/configure-bigquery
+
+      - name: Collect and Store Step Status
+        run: yarn actions-steps-error-check
+        working-directory: qa-standards-dashboard-data
+        env:
+          RUN_ID: ${{ github.run_id }}
+          ATTEMPT_NUMBER: ${{ github.run_attempt }}
+          MAIN_BRANCH: false
```

Comment:
Not exactly sure what `MAIN_BRANCH` is for, but I'm guessing it's for keeping track of if a branch is `main`? If so, you may want to make that dynamic, since this will still run on `main` based off the `if` statement above. Maybe something like `MAIN_BRANCH: ${{ github.ref == 'refs/heads/main' }}`

Reply:
This was more to track whether this comes from the CI workflow vs the full E2E run workflow. More or less, we wanted to distinguish results that could be someone just beating up a test branch from the full run. I can probably add a distinction to label by workflow, and change the `MAIN_BRANCH` to how you have it here as your explanation definitely makes more sense.

Reply:
This seems like a good proposed change. Any time we can build in resiliency is probably a worthwhile suggestion, thanks Theo.

Reply:
That makes sense. I think adding a distinction to label by workflow sounds like a good idea. That would also make it expandable if you wanted to track more workflows in the future too